java/src/main/java/org/opensky/api/OpenSkyApi.java

```diff
@@ -33,6 +33,8 @@
 	private static final String API_ROOT = "https://" + HOST + "/api";
 	private static final String STATES_URI = API_ROOT + "/states/all";
 	private static final String MY_STATES_URI = API_ROOT + "/states/own";
+	
+	private static HttpHost proxy;
```

Comment:
just for curiosity, why don't you use the proxied jvm.
As an example question from stack overflow
http://stackoverflow.com/questions/120797/how-do-i-set-the-proxy-to-be-used-by-the-jvm

in that way you dont have to separate the proxied call from the non-proxied one